conf/messages

```diff
@@ -57,7 +57,7 @@ vatDetails.vatOptOut.link = Opt out of Making Tax Digital for VAT
 vatDetails.vatOptOut.info = You cannot opt out if your taxable turnover has been above £85,000 since 1 April 2019.
 vatDetails.vatOptOut.pending = Your request to opt out is being processed.
 vatDetails.mtdSignup.link = Sign up for Making Tax Digital for VAT
-vatDetails.mtdSignup.info = If your taxable turnover exceeds the VAT threshold, you must sign up to Making Tax Digital for VAT.
+vatDetails.mtdSignup.info = You must sign up to Making Tax Digital for VAT if you''re not exempt from VAT, and your taxable turnover exceeds the £85,000 threshold.
```

Comment:
Can you use a smart apostrophe here instead of two straight ones. It renders slightly differently and you don't have to escape it. We should have replaced all of those a while ago (looks like a few are still lingering around unfortunately).